src/EditorFeatures/Core/DocumentationComments/DocumentationCommentSuggestion.cs

```diff
@@ -65,26 +65,39 @@ public override Task OnProposalUpdatedAsync(SuggestionSessionBase session, Propo
             return Task.CompletedTask;
         }
 
-        public async Task TryDisplaySuggestionAsync(CancellationToken cancellationToken)
+        public async Task<bool> StartSuggestionSessionAsync(CancellationToken cancellationToken)
```

Comment:
doc what the bool represents, and waht teh caller should do it it gets true vs false back.